test/parser/parser_test.cpp

```diff
@@ -1,3 +1,4 @@
+#include <parser/expression/aggregate_expression.h>
```

Comment:
Maybe move this to line 7 to match the other includes in style.

Reply:
Got it. This is auto-generated by CLion.